whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapObjectMapperTest.java

```diff
@@ -591,14 +599,49 @@ public void help() {
         assertThat(result.getPort43(), is("whois.ripe.net"));
     }
 
+    @Test
+    public void abuse_validation_failed() {
+        when(abuseContact.isSuspect()).thenReturn(true);
+        when(abuseContact.getOrgId()).thenReturn(ciString("ORG-NCC1-RIPE"));
+        when(abuseContact.getAbuseMailbox()).thenReturn(ciString("abuse@test.com"));
+        when(abuseContact.getNicHandle()).thenReturn(ciString("AB-TEST"));
+
+        final Autnum result = (Autnum) map(
+                RpslObject.parse("" +
+                    "aut-num:        AS102\n" +
+                    "as-name:        End-User-2\n" +
+                    "org:            ORG-NCC1-RIPE\n" +
+                    "admin-c:        AP1-TEST\n" +
+                    "tech-c:         AP1-TEST\n" +
+                    "abuse-c:        AB-TEST\n" +
+                    "notify:         noreply@ripe.net\n" +
+                    "mnt-by:         UPD-MNT\n" +
+                    "source:         TEST\n"
+                ),
+                RpslObject.parse(
+                    "role:           Abuse Contact\n" +
+                    "nic-hdl:        AB-TEST\n" +
+                    "mnt-by:         TEST-MNT\n" +
+                    "abuse-mailbox:  abuse@test.com\n" +
+                    "admin-c:        TP1-TEST\n" +
+                    "tech-c:         TP2-TEST\n" +
+                    "phone:          +31 12345678\n" +
+                    "source:         TEST"
+                ),
+                Optional.of(abuseContact)
+        );
+
+        assertThat(result.getRemarks().get(0).getDescription().get(0), is("Abuse-mailbox validation failed. Please refer to ORG-NCC1-RIPE for further information."));
```

Comment:
Can you add a negative test in a different test case? (i.e. the messages doesn't appear)

Reply:
I've added a bunch of test cases (message doesn't appear, responsible org not found, IT).